### PR TITLE
Added extracting example into readme

### PR DESCRIPTION
**Pull request by g3d**

### CODE REVIEW

**Comment by AlfonsoUceda** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -298,6 +298,36 @@ class ArticleRepository
 end
 ```
 
+You can also extract the common logic from your repository into a module to reuse it in other repositories. Here is pagination example:
```
Comment (joneslee85) [inline README.md]: s/is pag/is a pag/

**Comment by joneslee85** [comment]
:+1:

**Comment by jodosha** [comment]
@g3d Thank you. Merging :+1:

**Pull request merged**